### PR TITLE
add cloud_manager and refresh

### PR DESCRIPTION
**Pull request by durandom**

### CODE REVIEW

File: app/models/manageiq/providers/dummy_pro/inventory/persister/cloud_manager.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class ManageIQ::Providers::DummyPro::Inventory::Persister::CloudManager < ManagerRefresh::Inventory::Persister
+  include ManagerRefresh::Inventory::CloudManager
```
Comment (durandom) [inline app/models/manageiq/providers/dummy_pro/inventory/persister/cloud_manager.rb]: I'm creating a PR for core to add this module

File: app/models/manageiq/providers/dummy_pro/inventory/collector/cloud_manager.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class ManageIQ::Providers::DummyPro::Inventory::Collector::CloudManager < ManagerRefresh::Inventory::Collector
+  def connection
+    @connection ||= manager.connect
+  end
+
+  def vms
```
Comment (jameswnl) [inline app/models/manageiq/providers/dummy_pro/inventory/collector/cloud_manager.rb]: can you add a comment here about what this should do?

**Comment by jameswnl** [comment]
We'll need `TODO` comments at where implementor would need to do and what they need to do.
I'll merge first.

File: app/models/manageiq/providers/dummy_pro/inventory/collector/cloud_manager.rb (hunk 2 above)
Comment (durandom) [reply app/models/manageiq/providers/dummy_pro/inventory/collector/cloud_manager.rb]: yeah - definitely

**Pull request merged**